akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -575,10 +576,9 @@ private[remote] class EndpointManager(conf: Config, log: LoggingAdapter) extends
 
   val accepting: Receive = {
     case ManagementCommand(cmd) ⇒
-      val allStatuses = transportMapping.values map { transport ⇒
-        transport.managementCommand(cmd)
-      }
-      Future.fold(allStatuses)(true)(_ && _) map ManagementCommandAck pipeTo sender()
+      val allStatuses: immutable.Seq[Future[Boolean]] =
+        transportMapping.values.map(transport ⇒ transport.managementCommand(cmd))(scala.collection.breakOut)
+      akka.compat.Future.fold(allStatuses)(true)(_ && _) map ManagementCommandAck pipeTo sender()
```

Comment:
Yes, I thought that it would be fine to break it out as an `immutable.Seq` here anyway. That works fine for both the old `fold` and the new `foldLeft`